### ISSUE TITLE
[Data@TAMU] Community search page

### ISSUE THREAD

**Issue by devangm** [TAMULib/data-dspace-angular#10]
**URL:** https://data-pre.library.tamu.edu/communities/27abe8e6-f875-43e8-a681-8dbdaa5128c7/search
**WCAG 2.1 AA compliant? :** No
**Violation:** 1.3.1 Info and Relationships (A)
**Issues:**  Two H1 designations
**Notes:** Both "Research Projects" and "Search Results" are labeled H1
